### DISCUSSION TITLE
¿Porque me dio una rama master  y una main cuando subi mis archivos?

### DISCUSSION THREAD

**Discussion by jaaramirez** [LaunchX-InnovaccionVirtual/CursoIntroPython#296] [Q&A] [answered]
Hola buenas tardes, podrían ayudarme con una duda, cuando subí mis katas, se crearon 2 ramas, una que dice main y otra que dice master, la cuestion es que no puedo unirlas, segui las instrucciones pero no me deja a alguien le paso lo mismo?

**Comment by JavierMendez-Coder** [comment, marked as answer]
Buenas tardes. Para combinar ambas ramas puedes hacer un `merge` desde GitHub con un `pull request`.
![image](https://user-images.githubusercontent.com/99059210/154595716-39256b04-113e-4e5e-b78e-67d218d01cd0.png)
Y luego en el tab de `Pull requests` de tu repositorio buscar la solicitud y realizar el merge. Tal y como lo vimos en los videos sobre GitHub de Launch X.
### Explicación
Lo que sucede es que GitHub cambió el nombre por defecto de la rama a `main` en vez de `master`.
Entonces, cuando en Git utilizas:
```
$ git push origin master
```
Lo que sucede es que estas creando una rama nueva llamada `master` en tu repositorio remoto.
Puedes cambiar su nombre en `Opciones` > `Repositorios`, y ahí debería aparecer la opción para establecer el nombre por defecto que desees, para que no te vuelva a suceder esto.
